tests/config/functional/setup_cfg/issue_3630/1.cfg

```diff
@@ -0,0 +1,3 @@
+# Wrong header ?
+[tool.pylint.MASTER]
+init-hook="from pylint.config import find_pylintrc;"
```

Comment:
Shall we use something like `print()` so we can actually check the `init-hook` is executed?